Healthcare-Hospital-Management-System/Healthcare-Hospital-Management-System/Models/Doctor.cs

```diff
@@ -0,0 +1,21 @@
+
+namespace HealthcareHospitalManagementSystem.Models
+{
+    public class Doctor : Person
+    {
+        public string Specialization { get; set; }
+        public string Department { get; set; }
+
+        // Override the abstract method from Person
```

Comment:
Please try to leave comments when you want to explain "why" you did it, not "what". Your comments just duplicate code and with time it will be hard to support both code and comments. Please clean up comments